core/src/Kokkos_Macros.hpp

```diff
@@ -245,8 +222,8 @@
 #endif
 #endif
 
-#if (1700 > KOKKOS_COMPILER_INTEL)
-#error "Compiling with Intel version earlier than 17.0 is not supported."
+#if (1900 > KOKKOS_COMPILER_INTEL)
```

Comment:
I suppose that means there is no fine grain way to differentiate patch releases.  Is that right?

Reply:
There seems not to be, in source code, when using the 19.x line at least.